rpm/python-ramalama.spec

```diff
@@ -34,12 +36,15 @@ BuildRequires: golang
 BuildRequires: golang-github-cpuguy83-md2man
 BuildRequires: git-core
 BuildRequires: make
-BuildRequires: python%{python3_pkgversion}-devel
 BuildRequires: pyproject-rpm-macros
+BuildRequires: python%{python3_pkgversion}-argcomplete
```

Comment:
Some of these build requirements should be dumped it's gonna cause portability problems for this rpm. huggingface is an example.